src/de/hasait/cipa/activity/CipaFileResourceCleanup.groovy

```diff
@@ -0,0 +1,49 @@
+package de.hasait.cipa.activity
+
+import de.hasait.cipa.Cipa
+import de.hasait.cipa.CipaNode
+import de.hasait.cipa.jobprops.JobParameterContainer
+import de.hasait.cipa.jobprops.JobParameterContribution
+import de.hasait.cipa.jobprops.JobParameterValues
+import de.hasait.cipa.resource.CipaFileResource
+import de.hasait.cipa.resource.NodeCleanup
+
+/**
+ * Performs clean up of all file resources for parallel activities with respect to each node.
+ */
+class CipaFileResourceCleanup extends AbstractCipaBean implements NodeCleanup, JobParameterContribution {
+
+    static final String PARAM___CLEANUP_RESOURCES = "CLEANUP_FILE_RESOURCES"
+
+    private boolean cleanUp
```

Comment:
I would skip the param and boolean completely. Disabling of cleanup is a rare case for error anaylsis and should be then done by not adding this bean - see my comment for addStandardBeans.